app/controllers/results_controller.rb

```diff
@@ -286,6 +307,27 @@ def records
     SQL
   end
 
+  private def compute_slim_or_separate_records(rows)
```

Comment:
Again, just wanna make sure: This is an exact copy of the helper function, right?

Reply:
yeah